.vscode/cspell.json

```diff
@@ -135,6 +136,8 @@
                 "securestring",
                 "Segoe",
                 "setvariable",
+                "SKUs",
```

Comment:
Is cspell case sensitive? @hallipr

Reply:
it's not

Reply:
Thanks for the confirmation. So we only need to keep one.